lib/vanity/frameworks/rails.rb

```diff
@@ -39,6 +39,10 @@ module UseVanity
       #     use_vanity { |controller| controller.params[:project_id] }
       #   end
       def use_vanity(symbol = nil, &block)
+        define_method :_vanity_experiments do
```

Comment:
If this is a publicly exposed method, it'd be better to remove the leading underscore.